CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.0
+* BREAKING: Updated to check against SHA-256 signature of request body (#47)
```

Comment:
My understanding is that both the sha-256 and the sha1 headers are still present in the Amazon response headers, and updating to this module doesn't involve any breaking changes, but maybe I'm missing something?

Reply:
You are correct, they still do have both headers. I figured if Amazon wants to use SHA-256 over SHA-1, they'll drop the latter at some point, and that would be a big behaviour change by itself (hence the major version bump).

However I'm not strongly rooted in this idea and open to other ideas too.

Reply:
I think semver is only referring to the impact of the module on a person's local codebase.  In the case of someone importing `alexa-verifier-middleware`, I don't see any changes that would be breaking on the code side.

Reply:
The reason why I'm bringing this up is the downside to bumping the major version is it results in fewer people updated. Bumping the minor number instead will result in more people adopting it quickly, which is nice for an API change that might someday break if/when Amazon removed the non-256 version of the signature header.

Reply:
You bring up a good point about version adoption. I'll make the changes later today.